tests/posix/common/testcase.yaml

```diff
@@ -1,32 +1,40 @@
 common:
-  filter: not CONFIG_NATIVE_LIBC
-  platform_exclude:
-    - native_posix
-    - native_posix/native/64
   tags: posix
   min_ram: 64
   timeout: 240
+  # 1 tier0 platform per supported architecture
+  integration_platforms:
```

Comment:
adding all those platforms into integration_platforms defeats the purpose, you can just get rid of integration_platforms completely in this file and get the same result. You can also use

```
  platform_key:
    - arch
    - simulation
 ```

Reply:
Fixed!